src/app/models/sap_model.php

```diff
@@ -100,6 +100,23 @@ public function get_submission_details($submission_id) {
         return $row;
     }
 
+    public function get_user_scores(){
+    	$stmt = $this->db_lib->prepared_execute(
+            $this->DB->sap,
+	        "SELECT s.`email`, s.`score`
```

Comment:
Oh, more importantly, `prepared_execute` isn't necessary. You can just call `$this->DB->sap->query('<SQL>');` of which you can find examples.